docs/data/date-pickers/custom-components/custom-components.md

```diff
@@ -80,3 +80,14 @@ This can be used in combination with `componentsProps`.
 In the example below, the actions are the same as in the section above, but they are rendered inside a menu:
 
 {{"demo": "ActionBarComponent.js"}}
+
+## Paper content
+
+The paper content is available on all desktop picker components.  
+It adds a flexible way to extend what is rendered in the picker paper.  
+You can provide any custom component to this slot as long as it passes `children` down maintaining original picker behavior.  
+In the examples below we provide ways of implementing date range shortcuts using this slot.
+
+{{"demo": "PaperContentComponent.js"}}
```

Comment:
For Pro demo you can disable ad, avoiding the following banner to appear by using `"disableAd": true`
![image](https://user-images.githubusercontent.com/45398769/186159123-eeddf753-bf39-44b7-8ea5-677fe8a124eb.png)

And the code is too large to display a useful preview, so you can hide it with `"defaultCodeOpen": false`
```suggestion
{{"demo": "PaperContentComponent.js", "defaultCodeOpen": false, "disableAd": true}}
```